.zuul.d/e2e-prepare.yml

```diff
@@ -0,0 +1,27 @@
+---
+- hosts: primary
+  gather_facts: true
+  tasks:
+    - name: Prepare workspace
+      ansible.builtin.include_role:
+        name: prepare-workspace
+    - name: Create zuul-output directory
+      ansible.builtin.file:
+        path: "{{ ansible_user_dir }}/zuul-output/logs"
+        state: directory
+        mode: 0755
+    - name: Install required packages
+      become: true
+      ansible.builtin.package:
+        name:
+          - make
+          - python3-pip
+    - name: Install requirements
+      ansible.builtin.shell:
+        chdir: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}"
+        cmd: |
+          git clone https://github.com/openstack-k8s-operators/install_yamls ~/install_yamls
```

Comment:
install_yamls is already clonned in {{ ansible_user_dir }}/{{ zuul.project.src_dir }}/github.com/openstack-k8s-operators/install_yamls, we can re-use that.